### DISCUSSION TITLE
Add "Hide diff highlights" toggle

### DISCUSSION THREAD

**Discussion by LegendM-P** [ReviewNB/support#119] [Ideas]
Hey, in my team we use ReviewNB a lot, but there is one thing that can be super annoying and that is the highlighting of changes for new notebooks. 

Many times our notebooks are one-off reports which can be quite big in size. The problem is that during the review process because all the lines are new, all the elements of the notebook are highlighted as light green. 

**Can I please request a toggle to hide the highlights?** 
It would make the notebooks so much more readable during their first PR. 

Thanks!

**Comment by amit1rrr** [comment]
Thanks @iTechMichalisPalaiokostas for bringing this up. Quick question - Instead of a toggle switch would it be better to always remove green highlighting for newly added notebooks? In other words, is there a use case where the green highlighting on newly added notebook is useful?

**Reply by LegendM-P** [reply]
Hmm, I personally can't think of any cases, but then again some people might like the colouring, hence the toggle. 😛 
In our case, removing it completely for new notebooks would make sense.

**Comment by amit1rrr** [comment]
> Instead of a toggle switch would it be better to always remove green highlighting for newly added notebooks?

@LegendMichalisPalaiokostas - Just fyi - we've released this change last year :)
